docs/content/tools/acquia-cli.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Acquia CLI Commands"
+aliases:
+  - /en/master/tools/acquia-drush/
+  - /tools/acquia-drush/
+---
+
+
+## Using Acquia Cloud CLI Commands
+
+The [Acquia Cloud CLI](https://github.com/typhonius/acquia_cli) tool allows you 
+to use all features of the Acquia Cloud API v2 on the command line.  
+
+Commands are pre-installed in `cli` container. Note: required `docksal/cli` version: 2.11.0+.
+
+
+## Configuration
+
+Follow the [official docs](https://cloud.acquia.com/#/profile/tokens) to 
+generate your Acquia Cloud API Token.  
+
+Add your secret and key to `$HOME/.docksal/docksal.env`:
+
+```
+...
+SECRET_ACQUIACLI_SECRET="user@example.com"
+SECRET_ACQUIACLI_KEY="rSxVZN35bo4jTuncGS+CiKdmhxLPL0BaPuyOvf510sKb6c4ycxZsfFpC/vjJgaMvGvpKi06iHoqX"
+...
+```
+
+Refresh `cli` service configuration with `fin project start`. This will inject the values into `cli`
+which will be used for authentication for the Acquia Cloud CLI.
+
+Note: It is also possible to add/override these values via `.docksal/docksal.env` and `.docksal/docksal-local.env` at the project level. Keep in mind, `.docksal/docksal.env` is a shared configuration file and should be committed to git. `.docksal/docksal-local.env`, on the other hand, can be used for local overrides and should be excluded from git. As such, it is best to use `.docksal/docksal-local.env` for any personal keys and tokens configured at the project level.
```

Comment:
Let's not suggest using `.docksal/docksal.env` for secrets at all.
`.docksal/docksal-local.env` is OK.